lxd/api_project.go

```diff
@@ -867,7 +867,7 @@ func projectDelete(d *Daemon, r *http.Request) response.Response {
 			return fmt.Errorf("Fetch project %q: %w", name, err)
 		}
 
-		empty, err := projectIsEmpty(ctx, project, tx)
+		empty, err := projectIsEmpty(ctx, project, tx, true)
```

Comment:
if there isnt anything on disk that contains the project name for cached images then i think we can allow rename when there are cached images.